scripts/configure_dev.sh

```diff
@@ -14,9 +14,16 @@ function install_or_upgrade {
 }
 
 if [ "${OS}" = "linux" ]; then
+    SUDO_INSTALLED=0
+    dpkg -l sudo >/dev/null 2>&1 || SUDO_INSTALLED=$?
```

Comment:
`dpkg -l sudo` exits with status 0 if the `sudo` package was removed (i.e., `dpkg -l sudo` prints a status line starting with `rc`).

Reply:
A fresh container was always in my mind but you're right since this would generally run in a developer env.